QuantConnect.BybitBrokerage/BybitBrokerage.Brokerage.cs

```diff
@@ -198,10 +198,9 @@ public override bool CancelOrder(Order order)
                 return;
             }
 
-            if (order.Status is OrderStatus.Canceled or OrderStatus.CancelPending)
+            if (order.Status is OrderStatus.Canceled)
             {
-                OnMessage(new BrokerageMessageEvent(BrokerageMessageType.Warning, -1,
-                    "Order already canceled or cancellation submitted"));
+                OnMessage(new BrokerageMessageEvent(BrokerageMessageType.Warning, -1, "Order already canceled"));
```

Comment:
Thank you! 👍  Yes, it's Lean transaction handler that set's the CancelPending state before sending the order cancel request to the brokerage https://github.com/QuantConnect/Lean/blob/master/Engine/TransactionHandlers/BrokerageTransactionHandler.cs#L477